swift_scality_backend/diskfile.py

```diff
@@ -331,8 +331,6 @@ class DiskFileReader(object):
 
     :param filesystem: internal file system object to use
     :param name: object name
-    :param obj_size: on-disk size of object in bytes
```

Comment:
Can't the arguments be actually removed from the `__init__` argument list?
If not, I'd leave them in the docstring, but mention they aren't used.